src/main.rs

```diff
@@ -61,6 +61,10 @@ pub struct RenderCmdOpts {
     /// Print rendered manifests to stdout
     #[structopt(short, long)]
     stdout: bool,
+
+    /// Run `helm dependencies update` before rendering deployments
+    #[structopt(short, long)]
+    update_dependencies: bool,
```

Comment:
Is there a case where we don't want to update the dependencies? I reckon that is why this is set to a `bool`

Reply:
Yes, by default it is false as this is often not required.

Reply:
It is only useful once you start to render charts which have remote sub-charts as dependencies.